HalloWing_Jump_Scare_Trap/hallowing_jump_scare_trap.py

```diff
@@ -0,0 +1,152 @@
+# HalloWing Jump Scare Trap
+# use PIR sensor, speaker, and servo
+import time
+import array
+import math
+import board
+import displayio
+import pulseio
+import digitalio
+import touchio
+import audioio
+# Setup LED and PIR pins
+LED_PIN = board.D13  # Pin number for the board's built in LED.
+PIR_PIN = board.SENSE   # Pin port connected to PIR sensor output wire.
+# Setup digital input for PIR sensor:
+pir = digitalio.DigitalInOut(PIR_PIN)
+pir.direction = digitalio.Direction.INPUT
+# Setup digital output for LED:
+led = digitalio.DigitalInOut(LED_PIN)
+led.direction = digitalio.Direction.OUTPUT
+# Setup servo
+servo = pulseio.PWMOut(board.D4, frequency=50)
+servo_min = 0.6
+servo_mid = 1.5
+servo_max = 2.45
+servo_time = 2.0
+# Setup cap touch button
+ready_button = touchio.TouchIn(board.TOUCH1)
+# Create a function to simplify setting PWM duty cycle for the servo:
+def servo_duty_cycle(pulse_ms, frequency=50):
```

Comment:
can you use the adafruit_motor library instead? it does all this servo stuff for ya!

Reply:
I think I had it for motor library at first and then switched it for some reason I'm now forgetting. I'll switch it back to that and make sure everything is still working.

Reply:
thanks  :)

Reply:
I must have imagined all that. It works great w the motor library code, I'll clean it up and re-commit.